config/config_params.go

```diff
@@ -118,7 +118,7 @@ type Config struct {
 	BPFEnabled                         bool           `config:"bool;false"`
 	BPFDisableUnprivileged             bool           `config:"bool;true"`
 	BPFLogLevel                        string         `config:"oneof(off,info,debug);off;non-zero"`
-	BPFDataIfacePattern                *regexp.Regexp `config:"regexp;^((en|wl|ww|sl|ib)[opsx].*|(eth|wlan|wwan).*|tunl0$|wireguard.cali$)"`
+	BPFDataIfacePattern                *regexp.Regexp `config:"regexp;^((en|wl|ww|sl|ib)[opsx].*|(eth|wlan|wwan).*|tunl0$|vxlan.calico$|wireguard.cali$)"`
```

Comment:
BTW, I think this is mixing stuff that is reasonably user-configurable (`en|wl|ww|sl|ib)[opsx].*|(eth|wlan|wwan)`), and stuff that is actually required for our implementation to work (`tunl0$|vxlan.calico$|wireguard.cali`).  We should probably separate those out.